examples/java/pom.xml

```diff
@@ -89,6 +89,18 @@
       </dependencies>
     </profile>
 
+    <!-- Include the Apache Gearpump (incubating) runner with -P gearpump-runner -->
+    <profile>
+      <id>gearpump-runner</id>
```

Comment:
Take a look in this file for the `jenkins-precommit` bits. You can add an execution for the Gearpump runner.